components/match2/wikis/ageofempires/legacy/match_group_legacy_default.lua

```diff
@@ -18,42 +16,10 @@ local MatchGroupLegacyDefault = Class.new(MatchGroupLegacy)
 
 ---@return table
 function MatchGroupLegacyDefault:getMap()
-	local map = {
+	return {
 		['$notEmpty$'] = 'map$1$',
-		map = 'map$1$',
-		mode = 'map$1$mode',
-		winner = 'map$1$win',
-		civs1 = 'map$1$p1civ',
-		civs2 = 'map$1$p2civ',
-		vod = 'vodgame$1$',
-		date = 'date$1$',
+		['$parse$'] = 'map$1$'
```

Comment:
What's $parse$?

Reply:
It will json.parse instead of mapping map args